client/wfprev-war/src/main/angular/src/app/components/evaluation-criteria-dialog/evaluation-criteria-dialog.component.ts

```diff
@@ -0,0 +1,382 @@
+import { TextFieldModule } from '@angular/cdk/text-field';
+import { CommonModule } from '@angular/common';
+import { Component, Inject } from '@angular/core';
+import { FormBuilder, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MAT_DIALOG_DATA, MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { forkJoin } from 'rxjs';
+import { ConfirmationDialogComponent } from 'src/app/components/confirmation-dialog/confirmation-dialog.component';
+import { EvaluationCriteriaCodeModel, EvaluationCriteriaSummaryModel, Project, WuiRiskClassCodeModel } from 'src/app/components/models';
+import { CodeTableServices } from 'src/app/services/code-table-services';
+import { ProjectService } from 'src/app/services/project-services';
+import { Messages } from 'src/app/utils/constants';
+@Component({
+  selector: 'wfprev-evaluation-criteria-dialog',
+  standalone: true,
+  imports: [CommonModule, ReactiveFormsModule, TextFieldModule],
+  templateUrl: './evaluation-criteria-dialog.component.html',
+  styleUrl: './evaluation-criteria-dialog.component.scss'
+})
+export class EvaluationCriteriaDialogComponent {
+  codeTables = [
+    { name: 'evaluationCriteriaCodes', embeddedKey: 'evaluationCriteriaCode' },
+    { name: 'wuiRiskClassCodes', embeddedKey: 'wuiRiskClassCode'}
+  ];
+  messages = Messages;
+  criteriaForm!: FormGroup;
+  evaluationCriteriaCode: EvaluationCriteriaCodeModel[] = [];
+  wuiRiskClassCode: WuiRiskClassCodeModel[] = [];
+  mediumFilters: any[] = [];
```

Comment:
Use types rather than `any`. If these are actually undetermined types, it'd be better to define a type with something you can explicitly cast too

Reply:
changed to EvaluationCriteriaCodeModel[] = []